src/skills/formations/StopFormation.cpp

```diff
@@ -199,25 +197,25 @@ std::vector<std::vector<Vector2>> StopFormation::getStopPositions() {
 }
 
 bool StopFormation::positionShouldBeAvoided(Vector2 pos) {
-    return (pos.dist(ball->pos) < 0.9 || !world::field->pointIsInField(pos, 0.0));
+    return (pos.dist(ball->pos) < 0.9 || !field->pointIsInField(pos, 0.0));
 }
 
 std::vector<Vector2> StopFormation::getProperPositions(int amount) {
     std::vector<Vector2> properPositions;
-    auto field = world::field->get_field();
```

Comment:
This function definitely shows why field needs a refactor (look at the amount of unnecesarry copying here) but that's outside of this pr's scope.